Readme.md

```diff
@@ -1,21 +1,70 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128554121/18.1.10%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4837)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [HomeController.cs](./CS/Q515371/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/Q515371/Controllers/HomeController.vb))
-* [TreeListVirtualModeHelper.cs](./CS/Q515371/Models/TreeListVirtualModeHelper.cs) (VB: [TreeListVirtualModeHelper.vb](./VB/Q515371/Models/TreeListVirtualModeHelper.vb))
-* [_TreeListPartial.cshtml](./CS/Q515371/Views/Home/_TreeListPartial.cshtml)
-* [Index.cshtml](./CS/Q515371/Views/Home/Index.cshtml)
-<!-- default file list end -->
-# TreeList - How to load data from a database in virtual mode and implement drag-and-drop
+# Tree List for ASP.NET MVC - How to load data from a database in virtual mode and implement drag-and-drop functionality
```

Comment:
If "Tree List" refers to the extension name, then shouldn't it be "TreeList"?

Reply:
We use Tree List in docs and site as a marketing name. I think it is ok in an example name